doc/source/docs/user_guide/projections.rst

```diff
@@ -268,10 +268,10 @@ including their EPSG codes and proj4 string definitions.
 
 Next to the EPSG code mentioned above, there are also other ways to specify the
 CRS: an actual ``pyproj.CRS`` object, a WKT string, a PROJ JSON string, etc.
```

Comment:
Can we also link `pyproj.CRS`? Here and in other places as well.

Reply:
Just added them in the recent [commit](https://github.com/geopandas/geopandas/pull/1931/commits/c7dc9e3a74a660020f9de5503609a73d98d2268c).